larq/layers.py

```diff
@@ -1321,3 +1333,46 @@ def __init__(
             implementation=implementation,
             **kwargs
         )
+
+
+@utils.register_keras_custom_object
+class QuantDepthwiseConv2D(QuantizerBase, tf.keras.layers.DepthwiseConv2D):
```

Comment:
Could you add some docstrings to similar to https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/layers/convolutional.py#L1686-L1747